src/serac/numerics/functional/tests/functional_basic_h1_scalar.cpp

```diff
@@ -24,6 +24,34 @@
 using namespace serac;
 using namespace serac::profiling;
 
+template <int dim>
+struct TestThermalModelOne {
```

Comment:
Can these have a more descriptive name or a comment of what the difference between model one and two?